src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  people: Person[];
+  personId: string;
+};
+
+const findParent = (
+  array: Person[], parentName: string | null,
+): Person | null => {
+  return array.find(parent => parent.name === parentName) || null;
+};
+
+export const PeopleTable: React.FC<Props> = ({ people, personId }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            sex,
+            born,
+            died,
+            motherName,
+            fatherName,
+            slug,
+          } = person;
+
+          const mother = findParent(people, motherName);
+          const father = findParent(people, fatherName);
```

Comment:
Better prepare people array to work with it after we receive this data from server
Now you'll call this function on every PeopleTable render, better to do it once and work with prepared data